### PR TITLE
Update tmz.com extractor tests

### PR DESCRIPTION
**Pull request by diegorodriguezv**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

---

### Description of your *pull request* and other information

Fix the tests for the tmz.com extractor.
```
$ python -m unittest discover -k TMZ
...
Ran 10 tests in 5.634s

OK
```
They where all failing.
The extractor itself is working fine.

### CODE REVIEW

**Comment by pukkandan** [comment]
Pls see https://github.com/yt-dlp/yt-dlp/pull/3549#issuecomment-1108589347

**Comment by diegorodriguezv** [comment]
The comment @pukkandan is refering to is:

> The test conditions was recently made stricter. Because of this, many existing extractor tests currently fail. We generally don't bother fixing them until the extractor is being modified. Did you come across this when working on this extractor? If so, you can put this patch as part of the relevant PR
I was planning on fixing it but since it is working okay I ran the tests and saw they needed updating. I can see why in a project like this having some tests fail to save developer time makes sense. I've been working using TDD and having green tests all the time lets you focus on just the task at hand but more importantly it lets you create new features and know that you are not breaking anything else. That is why immediately jump to fix the tests.

**Comment by pukkandan** [comment]
I don't actually mind merging this. But if you found this while working on the extractor, it is easier to just add this commit to that PR

**Comment by diegorodriguezv** [comment]
I was going to fix it but it needed no fixing. I tested several new pages (videos) at tmz.com and it worked flawlessly. But the tests weren't working so I fixed them.
I would very much like this PR merged. In fact that is why I decided to contribute here instead of youtube-dl (took a year to merge my PR) or yt-dlc (died).
Besides I saw that "Tests: passing" on the readme and my heart fluttered.

File: yt_dlp/extractor/tmz.py [hunk 1]
```diff
@@ -133,7 +181,9 @@ def _real_extract(self, url):
         if not jsonld or "url" not in jsonld:
             # try to extract from YouTube Player API
             # see https://developers.google.com/youtube/iframe_api_reference#Video_Queueing_Functions
-            match_obj = re.search(r'\.cueVideoById\(\s*(?P<quote>[\'"])(?P<id>.*?)(?P=quote)', webpage)
+            match_obj = re.search(
+                r'\.cueVideoById\(\s*(?P<quote>[\'"])(?P<id>.*?)(?P=quote)', webpage
+            )
```
Comment (pukkandan) [inline yt_dlp/extractor/tmz.py:186]:
```suggestion
            match_obj = re.search(r'\.cueVideoById\(\s*(?P<quote>[\'"])(?P<id>.*?)(?P=quote)', webpage)
```

File: yt_dlp/extractor/tmz.py [hunk 2]
```diff
@@ -142,7 +192,8 @@ def _real_extract(self, url):
             if blockquote_el:
                 matches = re.findall(
                     r'<a[^>]+href=\s*(?P<quote>[\'"])(?P<link>.*?)(?P=quote)',
-                    blockquote_el)
+                    blockquote_el,
+                )
```
Comment (pukkandan) [inline yt_dlp/extractor/tmz.py:196]:
```suggestion
                    blockquote_el)
```

**Review by pukkandan — CHANGES_REQUESTED** [review]

File: yt_dlp/extractor/tmz.py [hunk 3]
```diff
@@ -108,21 +122,55 @@ class TMZIE(InfoExtractor):
...
+                    "esnews",
+                    "boxing",
+                    "mma",
+                    "sports interviews",
+                    "es news",
+                    "boxing news",
+                    "mma news",
+                    "boxing interviews",
+                    "mma interviews",
+                ],
```
Comment (pukkandan) [inline yt_dlp/extractor/tmz.py:151]:
```suggestion
                "tags": "count:10",
```

**Comment by pukkandan** [comment]
> Besides I saw that "Tests: passing" on the readme and my heart fluttered.

This is referring to core tests

**Pull request merged**